src/Commands/ElasticsearchIndexCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Rapidez\Core\Commands;
+
+use Carbon\Carbon;
+use Exception;
+use Rapidez\Core\Commands\InteractsWithElasticsearchCommand;
+
+abstract class ElasticsearchIndexCommand extends InteractsWithElasticsearchCommand
+{
+    /**
+     * Index all items in all stores
+     *
+     * @param  string $indexName
+     * @param  (callable(object): array)|array $data
+     * @param  (callable(object): array)|array $values
+     */
+    public function indexAll($indexName, $data, $values)
```

Comment:
I'm not quite convinced of the function name yet, what "All" does this index if we have not read this code?

Reply:
How about `indexAllStores`? 😇